common/src/main/java/org/wildfly/httpclient/common/Version.java

```diff
@@ -20,12 +20,18 @@
 /*
  * Versioning enum for HttpHandler implementations.
  *
+ * TODO: due to the way EENamespaceInteroperability.createInteroperabilityHandler(HttpHandler...) works,
```

Comment:
Yes, I see your point. It is the drawback of introducting the handler version after protocol version is already at number 2. I'm still open to suggestions if you would do it differently.

Reply:
I'm thinking about it...